crates/bevy_dev_tools/src/ci_testing.rs

```diff
@@ -2,15 +2,14 @@
 
 use bevy_app::{App, AppExit, Update};
 use bevy_ecs::{
-    entity::Entity,
-    prelude::{resource_exists, Resource},
-    query::With,
-    schedule::IntoSystemConfigs,
-    system::{Local, Query, Res, ResMut},
+    entity::Entity, event::Event, prelude::Resource, query::With, system::Local, world::World,
```

Comment:
Can this be simplified to just import from the prelude?

```suggestion
    prelude::*,
```